content/en/topics/azure-permissions.md

```diff
@@ -17,6 +17,8 @@ Microsoft.Network/loadBalancers/write
 Microsoft.Network/loadBalancers/backendAddressPools/read
 Microsoft.Network/loadBalancers/backendAddressPools/write
 Microsoft.Network/loadBalancers/backendAddressPools/delete
+Microsoft.Network/loadBalancers/inboundNatRules/join/action
```

Comment:
Is this required?

Reply:
Better to have. the permission is not used by cloud provider, but if customer setup nat rules outside, then cloud provider would need the permission to PUT LB.

Reply:
when i looked through the code, it seemed possible that this might be needed. it wasn't eniterly clear to me though because iirc, some of the calls where happening in the azure sdk and i couldn't see exactly where they got called.